### DISCUSSION TITLE
FirebaseError: Firebase: No Firebase App

### DISCUSSION THREAD

**Discussion by MattHeslington** [gladly-team/next-firebase-auth#288] [Q&A]
Hello,

Are any other users experiencing excessive 'FirebaseError: Firebase: No Firebase App '[DEFAULT]' has been created - call Firebase App.initializeApp()' errors? My app goes through stages of working normally, and then not working at all due to the above error.

I'm running just a basic auth setup as per the official example, no Firebase storage or Firestore instances.

Cheers, Matt

**Comment by arielmirra** [comment]
Same here! and if I try to initialize a firebase app the error changes to 'already created default app' (app/duplicated-app). Really frustrating.
